member_map.go

```diff
@@ -73,6 +71,19 @@ type Member struct {
 	// got suspicion message, that member can increments incarnation
 	Incarnation uint32
 }
+type AliveMessageID struct {
+	ID string
+}
+
+type AliveMessage struct {
+	ID AliveMessageID
+
```

Comment:
Chagne `ID AliveMessageID` to `ID string`

Reply:
OK